### PR TITLE
Update tests to use the robust function to hide cookies window

### PR DESCRIPTION
**Pull request by roienatan**

### CODE REVIEW

File: test/integration/utils.ts [hunk 1]
```diff
@@ -91,7 +75,7 @@ export async function gotoDaoPlugins(daoAddress: string): Promise<any> {
  * @param void
  * @returns Promise<void>
  */
-export const acceptCookies = async(): Promise<void> => {
+export const hideCookieAcceptWindow = async(): Promise<void> => {
```
Comment (dkent600) [inline test/integration/utils.ts:78]: the original code checked `!acceptCookiesButton.error`.  You sure you don't need that?  I wouldn't have put it in unless I found it was needed, though I no longer recall the reason.

Comment (roienatan) [reply test/integration/utils.ts:78]: I checked the tests locally and as you can see all the tests passed. It's not a must but I think the old function is not written in a stable way and can cause problems in the future.

Comment (dkent600) [reply test/integration/utils.ts:78]: I'm not questioning modifying the function, rather whether in doing so we should have deleted the check for `!acceptCookiesButton.error`.  I don't see any reason not to continue checking it.

Comment (dkent600) [reply test/integration/utils.ts:78]: But if you can continue that this works in all three use cases, then I can accept it as is:

1) travis
2) locally, headless
3) locally, not headless

Comment (roienatan) [reply test/integration/utils.ts:78]: I'm not sure what's the purpose of `.error` there but I couldn't find more places that `.error` is used in the tests.
By the way, I don't think `.error` is part of the `Jest.js` framework.

**Pull request merged**